jobs/yolov5-web-service/app.py

```diff
@@ -0,0 +1,63 @@
+import io
+import os
+import shutil
+import json
+from PIL import Image
+
+import torch
+import flask
+from flask import Flask, jsonify, url_for, render_template, request, redirect
+
+app = Flask(__name__)
+
+RESULT_FOLDER = os.path.join('static')
+app.config['RESULT_FOLDER'] = RESULT_FOLDER
+
+model_yolov5s = torch.hub.load('ultralytics/yolov5', 'yolov5s', pretrained=True).autoshape() 
+model_yolov5s.eval()
```

Comment:
Here we should find a way to `.load()` our own model and not the default one from yolov5.
It would be great if that model came from a volume also.